node/rust-client/src/lib.rs

```diff
@@ -163,9 +163,9 @@ pub fn init(level: Option<Level>, file_name: Option<&str>) -> Level {
 fn redis_value_to_js(val: Value, js_env: Env) -> Result<JsUnknown> {
     match val {
         Value::Nil => js_env.get_null().map(|val| val.into_unknown()),
-        Value::SimpleString(str) => js_env
-            .create_string_from_std(str)
-            .map(|val| val.into_unknown()),
+        Value::SimpleString(str) => Ok(js_env
+            .create_buffer_with_data(str.as_bytes().to_vec())?
```

Comment:
Why?

Reply:
we changed all the languages to return bytes